cmake/reports/CMakeLists.txt

```diff
@@ -14,5 +14,6 @@ foreach(report ram_report rom_report)
     --nm      ${CMAKE_NM}
     -o ${PROJECT_BINARY_DIR}
     DEPENDS ${logical_target_for_zephyr_elf}
+            $<TARGET_PROPERTY:zephyr_property_target,${report}_DEPENDENCIES>
```

Comment:
This comment refers to commit message, but placed here as I cannot comment directly i message:
```
 [nrf toup] cmake: allow projects to add report dependencies. 
```
should have been:
```
 [nrf fromlist] cmake: allow projects to add report dependencies. 
```
as we now have : https://github.com/zephyrproject-rtos/zephyr/pull/22006

Reply:
fixed